backend/metering_billing/webhooks.py

```diff
@@ -347,3 +347,54 @@ def subscription_cancelled_webhook(subscription, subscription_data=None):
                 )
             except Exception as e:
                 logger.error(e)
+
+
+def subscription_renewed_webhook(subscription, subscription_data=None):
+    from api.serializers.model_serializers import SubscriptionRecordSerializer
+    from metering_billing.models import WebhookEndpoint
+
+    if SVIX_CONNECTOR is not None:
+        endpoints = (
+            WebhookEndpoint.objects.prefetch_related("triggers")
+            .filter(triggers__trigger_name=WEBHOOK_TRIGGER_EVENTS.SUBSCRIPTION_RENEWED)
+            .distinct()
+        )
+        if endpoints.count() > 0:
+            svix = SVIX_CONNECTOR
+            now = str(now_utc())
+            payload = (
+                subscription_data
+                if subscription_data
+                else SubscriptionRecordSerializer(subscription).data
+            )
+            payload = make_all_decimals_floats(payload)
+            payload = make_all_dates_times_strings(payload)
+            response = {
+                "event_type": WEBHOOK_TRIGGER_EVENTS.SUBSCRIPTION_RENEWED,
+                "payload": payload,
+            }
+            event_id = (
+                slugify(str(subscription.customer))
+                + "_"
+                + slugify(str(subscription.billing_plan))
+                + "_"
+                + slugify(str(subscription.subscription_record_id.hex)[:50])
+                + "_"
+                + "created"
+            )
+
+            try:
```

Comment:
@djanda97 one thing to note is that I have added this try except block around the svix API call. This is because if svix API call fails this will make sure our webhook doesn't block the actual operation and only the webhook will fail to send